src/routes/github/configuration/github-configuration-get.ts

```diff
@@ -101,6 +101,7 @@ const getInstallationsWithAdmin = async (
 			...installation,
 			numberOfRepos,
 			isAdmin,
+			isSso: !!errors.find(err => err?.cause?.response?.headers["x-github-sso"]),
```

Comment:
It doesn't look too reliable tbh... Could we maybe modify isUserAdminOfOrganization to return some result object/enum that returns more details?

Reply:
Created a new Error instance `GithubClientSSOLoginError` and matched the type of error with this type